tests/instances.spec.ts

```diff
@@ -228,3 +230,21 @@ test("instance yaml edit", async ({ page }) => {
 
   await deleteInstance(page, instance);
 });
+
+test("instance terminal operations", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await visitAndStartInstance(page, instance);
+  await page.getByTestId("tab-link-Terminal").click();
+  await expect(page.locator(".xterm-screen")).toBeVisible(TIMEOUT);
+  const terminalInput = page.getByRole("textbox", { name: "Terminal input" });
+  await terminalInput.fill("lsb_release -a");
+  await terminalInput.press("Enter");
+  await expect(page.locator(".xterm-rows")).toContainText("jammy");
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  page.on("dialog", async (dialog) => {
+    expect(dialog.type()).toEqual("beforeunload");
+    await dialog.accept();
+  });
```

Comment:
Can you extend this a bit? I.e. assert, the dialogue is displayed on navigation to another tab. This might also allow for the removal of the eslint  ignore.